Haunted One.txt

```diff
@@ -0,0 +1,7 @@
+Name:Haunted One
+ManaCost:2 B
+Types:Legendary Enchantment Background
+S:Mode$ Continuous | Affected$ Creature.IsCommander+YouOwn | AddTrigger$ BecomesTapped | Description$ Commander creatures you own have “Whenever this creature becomes tapped, it and other creatures you control that share a creature type with it each get +2/+0 and gain undying until end of turn.”
+SVar:BecomesTapped:Mode$ Taps | ValidCards$ Card.Self | Execute$ TrigPump | TriggerDescription$ Whenever this creature becomes tapped, it and other creatures you control that share a creature type with it each get +2/+0 and gain undying until end of turn.
+SVar:TrigPump:DB$ PumpAll | ValidCards$ Card.sharesCreatureTypeWith | KW$ Undying | NumAtt$ 2 |

```

Comment:
don't forget `YouCtrl` or you buff opp cards too 😉 

pls remove extra pipe at end of line `|`

Reply:
Is this also buffing the commander? Seems like it would, but maybe it should be `ValidCards$ Card.Self,Card.sharesCreatureTypeWith` just to avoid some weird corner case